src/components/StoryCreationComponent.js

```diff
@@ -68,78 +52,122 @@ class StoryCreationComponent extends Component {
+          buttons: [
+            {
+              label: 'OK'
+            }
+          ]
+        });
+        return;
+      } else if (this.state.titleField.length < 1) {
+        confirmAlert({
+          title: 'Error!',
+          message: 'Please enter a story title.',
+          buttons: [
+            {
+              label: 'OK'
+            }
+          ]
+        });
+        return;
+      } else {
+        axios
+          .post(`${TRUTHTREE_URI}/api/stories`, {
+            author: this.state.authorField,
+            tags:
+              this.state.tagsField.length > 0
+                ? this.state.tagsField
+                : [this.state.tagsInputValue],
+            content: this.state.storyField
+          })
+          .then(function(response) {
+            console.log('saved successfully' + response);
+          });
+        confirmAlert({
+          title: 'Story submitted!',
+          message: 'Story is now pending review.',
+          buttons: [
+            {
+              label: 'Continue.'
+            }
+          ]
+        });
+      }
     } else {
-      /* {
-              storyTitle: this.titleField,
-              
-          }*/
+      if (this.state.storyTextOnly.length > this.state.storyMaxLength) {
+        alert('Story text is too long.');
+        return;
+      } else if (this.state.titleField.length < 1) {
+        alert('Please enter a story title.');
+        return;
+      } else {
+        axios
+          .post(`${TRUTHTREE_URI}/api/stories`, {
+            author: this.state.authorField,
+            tags:
+              this.state.tagsField.length > 0
+                ? this.state.tagsField
+                : [this.state.tagsInputValue],
+            content: this.state.storyField
+          })
+          .then(function(response) {
+            console.log('saved successfully' + response);
+          }); /*
+        this.props.dispatch({
+            type: 'CLOSE_STORY',
+            value: false
+        });*/
+        alert(
+          'Story submitted! Feel free to add a new one, or close the window.'
+        );
+      }
     }
   }
-  /*
-    deselectAttribute(attribute) {
-        let newArr = this.state.selectedAttributes;
-        let id = attribute[0];
-        for (let i = 0; i < newArr.length; i++) {
-            if (newArr[i][0] === id) {
-                _.remove(newArr, elem => {
-                    return elem === newArr[i];
-                });
-                this.setState({
-                    selectedAttributes: newArr
-                });
-                this.props.dispatch({
-                    type: 'CHANGE_ATTRIBUTE',
-                    value: newArr
-                });
-                return;
-            }
-        }
-    }
-    */
+
+  removeTag = tag => {
+    let newArr = this.state.tagsField;
+    newArr.splice(tag, 1);
+    this.setState({
+      tagsField: newArr
+    });
+  };
   render() {
     if (this.state.isLoaded) {
```

Comment:
`isLoaded` is not used anymore, you can remove this condition check

Reply:
it's being used. See bottom of code